packages/jest-snapshot/src/__tests__/SnapshotFile-test.js

```diff
@@ -99,4 +99,21 @@ describe('SnapshotFile', () => {
     expect(snapshotFile.get(SNAPSHOT)).toBe('"' + SNAPSHOT_VALUE + '"');
     snapshotFile.save();
   });
+
+  it('sorts snapshots by natural sort order', () => {
+    const fs = require('fs');
+    const snapshotFile = SnapshotFile.forFile(TEST_FILE);
+    ['test 2', 'test 11', 'test 1', 'test 10'].forEach(snapshot =>
+      snapshotFile.add(snapshot, SNAPSHOT_VALUE),
+    );
+    fs.writeFileSync.mockImplementation((path, content) => {
+      expect(content).toBe(
+        'exports[`test 1`] = `"bar"`;\n\n' +
+        'exports[`test 2`] = `"bar"`;\n\n' +
+        'exports[`test 10`] = `"bar"`;\n\n' +
+        'exports[`test 11`] = `"bar"`;\n',
+      );
+    });
+    snapshotFile.save();
```

Comment:
can you add `expect(fs.writeFileSync).toBeCalled()`? Otherwise if we change .save to not use `fs.writeFileSync` any more, this test will still pass and not test anything.

Instead, you can also test like this:

```
expect(fs.writeFileSync).toBeCalledWith([path, 'exports[…] = …']);
```

which is probably best :)